demos/simple-storage/dapp/package.json

```diff
@@ -35,6 +33,10 @@
     "prestart": "yarn install:web3api",
     "install:web3api": "npx web3api-worker-install ./public"
   },
+  "resolutions": {
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0"
+  },
```

Comment:
IIRC: since we are doing this is a monorepo, when we added the react dependencies in the react package, it wasn't working because it was taking the versioning from this project (dapp) so we had to create this resolutions attribute which tells yarn to use this versioning in this project

Reply:
Ah okay, makes sense